recipes/r-heatmap.plus/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = '1.3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-heatmap.plus
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: heatmap.plus_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/heatmap.plus_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/heatmap.plus/heatmap.plus_{{ version }}.tar.gz
+  sha256: c3f263e9055b9dbcfcaa387c7ef7f09fa17db8997982590a3d2b14554275ee67
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('heatmap.plus')"  # [not win]
+    - "\"%R%\" -e \"library('heatmap.plus')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=heatmap.plus
+  license: GPL
+  summary: Allows heatmap matrix to have non-identical X- and Y-dimensions.  Allows multiple
+    tracks of annotation for RowSideColors and ColSideColors.
+  license_family: GPL
```

Comment:
the license_file is here missing

Reply:
Well, it's just licensed under "GPL", so it's not really clear which one to pick.

Reply:
Is there a file included in the tarball?

Reply:
No.